src/main/resources/assets/forge/lang/en_US.lang

```diff
@@ -92,6 +92,38 @@ forge.configgui.allowEmissiveItems.tooltip=Allow item rendering to detect emissi
+forge.configgui.hud.allowHotbar=Allow Hotbar Rendering
+forge.configgui.hud.allowHotbar.tooltip=Toggle off to force disable hotbar rendering. Note that this config is applied on config saved, mods can still change hotbar rendering behaviour after that.
+forge.configgui.hud.allowCrosshairs=Allow Crosshairs Rendering
+forge.configgui.hud.allowCrosshairs.tooltip=Toggle off to force disable crosshairs rendering. Note that this config is applied on config saved, mods can still change crosshairs rendering behaviour after that.
+forge.configgui.hud.allowBossHealth=Allow Boss Health Rendering
+forge.configgui.hud.allowBossHealth.tooltip=Toggle off to force disable boss health rendering. Note that this config is applied on config saved, mods can still change boss health rendering behaviour after that.
+forge.configgui.hud.allowHealth=Allow Health Rendering
+forge.configgui.hud.allowHealth.tooltip=Toggle off to force disable health rendering. Note that this config is applied on config saved, mods can still change health rendering behaviour after that.
+forge.configgui.hud.allowArmor=Allow Armor Rendering
+forge.configgui.hud.allowArmor.tooltip=Toggle off to force disable armor rendering. Note that this config is applied on config saved, mods can still change armor rendering behaviour after that.
+forge.configgui.hud.allowFood=Allow Food Rendering
+forge.configgui.hud.allowFood.tooltip=Toggle off to force disable food rendering. Note that this config is applied on config saved, mods can still change food rendering behaviour after that.
+forge.configgui.hud.allowHealthMount=Allow Health Mount Rendering
+forge.configgui.hud.allowHealthMount.tooltip=Toggle off to force disable health mount rendering. Note that this config is applied on config saved, mods can still change health mount rendering behaviour after that.
+forge.configgui.hud.allowAir=Allow Air Rendering
+forge.configgui.hud.allowAir.tooltip=Toggle off to force disable air rendering. Note that this config is applied on config saved, mods can still change air rendering behaviour after that.
+forge.configgui.hud.allowExperience=Allow Experience Rendering
+forge.configgui.hud.allowExperience.tooltip=Toggle off to force disable experience rendering. Note that this config is applied on config saved, mods can still change experience rendering behaviour after that.
+forge.configgui.hud.allowJumpBar=Allow Jump Bar Rendering
+forge.configgui.hud.allowJumpBar.tooltip=Toggle off to force disable jump bar rendering. Note that this config is applied on config saved, mods can still change jump bar rendering behaviour after that.
+forge.configgui.hud.allowObjective=Allow Objective Rendering
+forge.configgui.hud.allowObjective.tooltip=Toggle off to force disable objective rendering. Note that this config is applied on config saved, mods can still change objective rendering behaviour after that.
```

Comment:
Align it with the aforementioned `Toggle off to disable rendering of the %s. Note that even though this config is applied when the config is saved to disk, mods can still override the behaviour after-the-fact.`